pyriemann/datasets/simulated.py

```diff
@@ -98,14 +99,15 @@ def make_gaussian_blobs(n_matrices=100, n_dim=2, class_sep=1.0, class_disp=1.0,
         Parameter controlling the separability of the classes.
     class_disp : float, default=1.0
         Intra dispersion of the points sampled from each class.
+    input_centers : ndarray, shape (2, n_dim, n_dim), default=None
```

Comment:
My advice is to call this parameter `centers`, like [make_blobs](https://scikit-learn.org/stable/modules/generated/sklearn.datasets.make_blobs.html).

Reply:
+1